src/adapter/src/coord/peek.rs

```diff
@@ -187,11 +188,8 @@ pub fn create_fast_path_plan<T: timely::progress::Timestamp>(
         if let Some((rows, ..)) = mir.as_const() {
             // In the case of a constant, we can return the result now.
             return Ok(Some(FastPathPlan::Constant(
-                rows.clone().map(|rows| {
-                    rows.into_iter()
-                        .map(|(row, diff)| (row, T::minimum(), diff))
-                        .collect()
-                }),
+                rows.clone()
+                    .map(|rows| rows.into_iter().map(|(row, diff)| (row, diff)).collect()),
```

Comment:
For some reason this `map` is needed so the iterator items are `(&Row, &Diff)` instead of `&(Row, Diff)`. If you know a cleaner way, then let me know.